library/src/scripts/content/translationGrid/TranslationGrid.tsx

```diff
@@ -28,11 +41,49 @@ export interface ITranslationGrid {
  */
 
 export function TranslationGrid(props: ITranslationGrid) {
-    const { data, inScrollingContainer = false } = props;
+    const id = useUniqueID("articleOtherLanguages");
+    const { data, inScrollingContainer = false, otherLanguages } = props;
+    const dateUpdated = "2019-10-08T13:54:41+00:00";
     const classes = translationGridClasses();
     const count = data.length - 1;
     const [translations, setTranslations] = useState(data);
     const translationKey = "newTranslation";
+    const currentLocale = "en";
+    let selectedIndex = 0;
+    const selectBoxItems: ISelectBoxItem[] = props.otherLanguages.map((data, index) => {
+        const isSelected = data.locale === currentLocale;
+        if (isSelected) {
+            selectedIndex = index;
+        }
+        return {
+            selected: isSelected,
+            name: data.locale,
+            icon: data.translationStatus === "not-translated" && (
+                <ToolTip
+                    label={
+                        <Translate
+                            source="This article was edited in source locale on <0/>. Edit this article to update its translation and clear this message."
+                            c0={<DateTime timestamp={props.dateUpdated} />}
+                        />
+                    }
+                    ariaLabel={"This article was editied in its source locale."}
```

Comment:
This should be describing what's the tool tip content is. In this case it would be "Warning Icon"